discord/message.py

```diff
@@ -518,6 +518,9 @@ def edit(self, **fields):
             If provided, the number of seconds to wait in the background
             before deleting the message we just edited. If the deletion fails,
             then it is silently ignored.
+        reason: Optional[str]
```

Comment:
This is kind of pointless. Would be better to revert your two commits.

I'm going to end up removing them from `send` as well since by design discord does not log bot deletes.